### PR TITLE
Fix constructors with multiple sets of arguments.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: files/en-us/web/api/passwordcredential/passwordcredential/index.html [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
+new PasswordCredential(htmlFormElement)</pre>
 
 <h3 id="Parameters">Parameters</h3>
 
 <p>Either of the following:</p>
 
 <dl>
   <dt><em>passwordCredentialData</em></dt>
-  <dd>A PasswordCredentialData dictionary containing the following fields:
+  <dd>An object with the following propeties:
```
Comment (teoli2003) [inline files/en-us/web/api/passwordcredential/passwordcredential/index.html:31]: nit properties:
```suggestion
  <dd>An object with the following properties:
```

**Review by teoli2003 — APPROVED** [review]
This is much better. One typo to fix and we can merge.

File: files/en-us/web/api/keyframeeffect/keyframeeffect/index.html [hunk 2]
```diff
@@ -18,54 +18,59 @@
...
-var keyframes = new KeyframeEffect(<em>sourceKeyFrames</em>);
+<pre class="brush: js">new KeyframeEffect(target, keyframes);
+new KeyframeEffect(target, keyframes, options)
+new KeyframeEffect(source)
 </pre>
 
 <h3 id="Parameters">Parameters</h3>
 
-<p>The first type of constructor (see above) creates a completely new {{domxref("KeyframeEffect")}} object instance. Its parameters are:</p>
+<p>The multi-argument constructors (see above) creates a completely new {{domxref("KeyframeEffect")}} object instance. Its parameters are:</p>
```
Comment (rachelandrew) [inline files/en-us/web/api/keyframeeffect/keyframeeffect/index.html:28]:
```suggestion
<p>The multi-argument constructor (see above) creates a completely new {{domxref("KeyframeEffect")}} object instance. Its parameters are:</p>
```

**Review by rachelandrew — APPROVED** [review]
just a small typo.

**Comment by jpmedley** [comment]
PTAL

**Pull request merged**